components/ExampleNav/ExampleNavContainer.coffee

```diff
@@ -4,6 +4,10 @@
 ExampleNav                     = require './ExampleNav'
 
 navs =
+  Dropdowns: [
+    'NavbarDropdownExamples'
```

Comment:
I think differently because of the previous merge conflicts. The Panels references were not in the router hence Git saw that it was deleted in the PR. Correct me if I'm wrong tho.